cli/elastic.py

```diff
@@ -45,3 +47,10 @@ def store_files(index: str, files: [str], cfg_name: str):
                 body=json.load(fh),
                 )
             )
+
+def store_dir(index: str, directory: str, cfg_name: str):
```

Comment:
use util.CliHints.existing_dir() as type (then the existence will be checked automatically)